src/project/irecentprojectscontroller.h

```diff
@@ -27,15 +27,18 @@
 #include "async/notification.h"
 
 namespace mu::project {
-class IRecentProjectsProvider : MODULE_EXPORT_INTERFACE
+class IRecentProjectsController : MODULE_EXPORT_INTERFACE
```

Comment:
Please fix #ifndef MU_PROJECT_IRECENTPROJECTSPROVIDER_H
Also, restore stub for this interface